porting/npl/freertos/include/npl/npl.h

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
Why isn't there a mynewt/include/npl/npl.h?

All platforms should use the exact same osal, including mynewt.

Reply:
As I wrote in description, the best would be if NimBLE defined its own interface for mutexes, semaphores etc. as true NPL and have all platforms implement this NPL, including Mynewt. This is not possible at the moment since it requires some more changes in NimBLE but it should be possible eventually. At the moment we need to use API imposed by Mynewt and just use this as NPL (or "NPL").